model/task_lifecycle.go

```diff
@@ -128,6 +128,10 @@ func SetActiveState(t *task.Task, caller string, active bool) error {
 		return nil
 	}
 
+	if err := UpdateBuildAndVersionStatusForTask(t); err != nil {
```

Comment:
What was the logic for moving this above the IsPartOfDisplay check? I would think it wouldn't work right until the display task status is updated at 136.

Reply:
yeah.